includes/Checker/Preparations/Use_Minimal_Theme_Preparation.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Preparations\Use_Minimal_Theme_Preparation
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Preparations;
+
+use WordPress\Plugin_Check\Checker\Preparation;
+use Exception;
+
+/**
+ * Class for the preparation step to force usage of a minimal theme.
+ *
+ * This ensures the plugin is checked as much in isolation as possible.
+ *
+ * @since n.e.x.t
+ */
+class Use_Minimal_Theme_Preparation implements Preparation {
+
+	/**
+	 * Theme slug / directory name.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	protected $theme_slug;
+
+	/**
+	 * Absolute path to themes root directory.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	protected $themes_dir;
+
+	/**
+	 * Sets the theme slug and themes root directory.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $theme_slug Slug of the theme to enforce.
+	 * @param string $themes_dir Optional. Absolute path to themes root directory, if not the regular wp-content/themes.
+	 */
+	public function __construct( $theme_slug, $themes_dir = '' ) {
+		$this->theme_slug = $theme_slug;
+		$this->themes_dir = $themes_dir;
+	}
+
+	/**
+	 * Runs this preparation step for the environment and returns a cleanup function.
+	 *
+	 * @since n.e.x.t
+	 *
```

Comment:
```suggestion
	 *
	 * @global array $wp_theme_directories
	 *
```

Add documentation for global variable.